lua/popup/init.lua

```diff
@@ -94,7 +94,18 @@ function popup.create(what, vim_options)
 
   if vim_options.line then
     -- TODO: Need to handle "cursor", "cursor+1", ...
-    win_opts.row = vim_options.line
+    if type(vim_options.line) == 'string' then
+      assert(string.find(vim_options.line,'^cursor') ~= nil, "Invalid option")
+      local line = vim.fn.getcurpos()[2]
+      if (vim_options.line):match "cursor%+(%d+)" then
+        line = line + tonumber((vim_options.line):match "cursor%+(%d+)")
```

Comment:
Can we write a util function that does transformation of `cursor+1` -> `1` so that we can re-use it? Or just a local function that we re-use.